src/Nethermind/Ethereum.Transition.Test/BerlinToLondonTests.cs

```diff
@@ -24,7 +24,7 @@
 namespace Ethereum.Transition.Test
 {
     [TestFixture]
-    [Parallelizable(ParallelScope.All)]
+    [Parallelizable(ParallelScope.None)]
```

Comment:
can't be parallel?

Reply:
When there is transition to London during tests (like here), there is a need to change Eip1559TransitionBlock in HeaderDecoder to decode London headers properly. In standard mode we are setting it in InitRlp (reading from genesis). In hive genesis is always Frontier and doesn't have this information. I'm setting it in BlockchainTestBase (lines 122-123) and unfortunately it is overwritten when running other tests in parallel

Reply:
And this tests are failing on hive and I haven't found solution yet